### PR TITLE
Cart logic hirin

### PR DESCRIPTION
**Pull request by roshirin**
- basic cart logic
- bug and code style fixes

### CODE REVIEW

File: src/context/StoreContextProvider.tsx [hunk 1]
```diff
@@ -39,6 +39,25 @@ export const StoreContextProvider: FC<Props> = ({ children }) => {
     });
   };
 
+  const changeCartProdQuantity = (id: string, delta: number) => {
+    setCartContents((currentContents) => {
+      // eslint-disable-next-line no-console
+      console.log(id, delta);
```
Comment (PavloReutskiy) [inline src/context/StoreContextProvider.tsx:45]: Maybe it's extra here

File: src/pages/Cart.tsx [hunk 2]
```diff
@@ -1,39 +1,71 @@
...
 
   const onCheckout = () => {
     setTimeout(() => {
       setShowModal(true);
     }, 100);
   };
 
+  const phoneIds = cartContents.map(({ id }) => id).join(',');
+
+  const getCartContents = useCallback(async() => {
```
Comment (PavloReutskiy) [inline src/pages/Cart.tsx:24]: Forgot about loading process and about Loader component

File: src/pages/Cart.tsx [hunk 3]
```diff
@@ -1,39 +1,71 @@
...
-            <Checkout
-              totalPrice={2000}
-              totalItems={3}
-              onCheckout={onCheckout}
-            />
-          </div>
+          {products.length > 0
+            ? (
+              <div className="cart">
+                <CartList products={products} />
```
Comment (PavloReutskiy) [inline src/pages/Cart.tsx:58]: What if products.length === 0?
Maybe you should render empty cart message

**Review by PavloReutskiy — APPROVED** [review]

File: src/pages/Cart.tsx (hunk 3 above)
Comment (roshirin) [reply src/pages/Cart.tsx:58]: This will be done

File: src/pages/Cart.tsx (hunk 2 above)
Comment (roshirin) [reply src/pages/Cart.tsx:24]: Loader is present down the code. But its logic needs to be adjusted for sure

**Pull request merged**